examples/vision/3D_image_classification.py

```diff
@@ -229,34 +195,17 @@ def plot_slices(num_rows, num_columns, width, height, data):
 )
 
 """
-## Preprocessing and data augmentation
+## Data augmentation
 
-CT scans store raw voxel intensity in Hounsfield units (HU). They range from
--1024 to above 2000 in this dataset. Above 400 are bones with different
-radiointensity, so this is used as a higher bound. A threshold between
--1000 and 400 is commonly used to normalize CT scans. The CT scans are
-also augmented by rotating and blurring. There are different kinds of
-preprocessing and augmentation techniques out there, this example shows a few
-simple ones to get started.
+The CT scans also augmented by rotating at random angles during training. Since there are
+no channels present in the data, a channel of 1 is added to perform 3D convolutions on the data.
```

Comment:
"Since the data is stored in rank-3 tensors of shape `(samples, height, width, depth)`, we add a dimension of size 1 at axis 4 to be able to perform 3D convolutions on the data. The new shape is thus `(samples, height, width, depth, 1)`."